producers/docker_trivy/main_test.go

```diff
@@ -10,15 +10,57 @@ import (
 
 	"github.com/stretchr/testify/assert"
 )
+func TestParseCombinedOut(t *testing.T) {
+	var results types.CombinedOut
+    combinedOutput := fmt.Sprintf("{\"ubuntu:latest\":%s,\"alpine:latest\":%s}",exampleOutput,exampleOutput)
```

Comment:
nit: indentation

Reply:
```suggestion
    combinedOutput := fmt.Sprintf(`{"ubuntu:latest":%s,"alpine:latest":%s}`, exampleOutput, exampleOutput)
```

Use backticks as string delimiters to avoid the need to escape double-quotes; separate function parameters with spaces